nltk/corpus/reader/conll.py

```diff
@@ -224,12 +224,13 @@ def _get_tagged_words(self, grid, tagset=None):
             pos_tags = [map_tag(self._tagset, tagset, t) for t in pos_tags]
         return list(zip(self._get_column(grid, self._colmap['words']), pos_tags))
 
-    def _get_iob_words(self, grid, tagset=None):
+    def _get_iob_words(self, grid, tagset=None, columns='chunk'):
+        columns = columns if isinstance(columns, list) else [columns]
```

Comment:
I would suggest to set `columns` to `None` rather then checking whether a list is passed, what if a `tuple` is passed or a `set` or any other `iterable`?

Also, this is a private method, so taking care that an argument could be an item or a list is not necessary. (It might be a good idea for a public API.)

I would prefer a clear solution that `columns` is an iterable.

Reply:
Updated!